state/accountsDB.go

```diff
@@ -1136,15 +1160,26 @@ func (adb *AccountsDB) SnapshotState(rootHash []byte) {
 	adb.waitForCompletionIfRunningInImportDB(stats)
 }
 
-func (adb *AccountsDB) markActiveDBAfterSnapshot(stats *snapshotStatistics, missingNodesCh chan []byte, errChan chan error, rootHash []byte, message string, epoch uint32) {
+func finishSnapshotOperation(
```

Comment:
if you attach this function to AccountsDB then you would have not needed to pass the trieStorageManager instance

Reply:
Done.